src/pages/MapPage.js

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+
+import Map from '../components/Map'
+
+// Mock location data
+const locations = [
+  {
+    id: 1,
+    lat: 40.1127151,
+    lng: -88.2314734,
+  },
+  {
+    id: 2,
+    lat: 40.1125785,
+    lng: -88.2287926,
+  },
+  {
+    id: 3,
+    lat: 40.112657,
+    lng: -88.2278543,
+  },
+  {
+    id: 4,
+    lat: 60.1125785,
+    lng: -88.2287926,
+  },
+  {
+    id: 5,
+    lat: 60.112657,
+    lng: -88.2278543,
+  },
+]
+
+// Mock cluster data
+const clusters = [
+  {
+    lat: 40.1127151,
+    lng: -88.2314734,
+    count: 1000,
+  },
+  {
+    lat: 60.1125785,
+    lng: -88.2278543,
+    count: 25,
+  },
+]
+
+const VISIBLE_CLUSTER_ZOOM_LIMIT = 12
+
+const DEFAULT_CENTER_LAT = 40.1125785
+
+const DEFAULT_CENTER_LNG = -88.2287926
+
+const DEFAULT_ZOOM = 1
+
+const DEFAULT_VIEW_STATE = {
+  center: [DEFAULT_CENTER_LAT, DEFAULT_CENTER_LNG],
+  zoom: DEFAULT_ZOOM,
+  bounds: null,
```

Comment:
Is `bounds` being updated here? If you expose `onViewChanged`, it would be the `Map` component's responsibility to update it.

`bounds` is necessary for the page to supply it when requesting clusters and locations.